release-team/role-handbooks/branch-manager/README.md

```diff
@@ -0,0 +1,183 @@
+    -   [Beta Build](#beta-build)
+    -   [RC Build](#rc-build)
+    -   [Official Build](#official-build)
+    -   [Debs and RPMs](#debs-and-rpms)
+    -   [Release](#release)
+    -   [Release Validation](#release-validation)
+-   [Branch Fast Forward](#branch-fast-forward)
+    -   [Code Slush, Freeze, Thaw](#code-slush-freeze-thaw)
+    -   [Reverts](#reverts)
+-   [Cherry Picks](#cherry-picks)
+
+---
+
+# References
+* [Old old anago doc](https://docs.google.com/document/d/1Qoqz5IZYBp6A-Q_R9CGhMAc358ykOiE49GXZU9r5usQ/edit#heading=h.s71iha1627td)
+* [Summary of discussion with Caleb about move from anago to gcb](https://groups.google.com/d/topic/kubernetes-milestone-burndown/YdHa51d95VI/discussion)
+* [Non-release-workflow-specific, generic gcbmgr documentation](https://github.com/kubernetes/release/blob/master/README.md)
+* [Patch release notes from exercising the new gcbmgr flow](https://docs.google.com/document/d/1x-GQDZpKk3WajtSnO0axDazE9Xs2mOSVgjziIuTWNO0/edit)
+
+# Pre-requirements
+* contact [Caleb Miles](mailto:calebmiles@google.com) (TODO: identify alternate/backup), identifying yourself as the incoming release branch manager for the current release team and requesting to be added to the special privilege group for the GCP project used to build the releases
+   * TODO: should the release lead also request some type of read access here for themselves?
+   * TODO: should the branch manager shadows also get at least read access at the beginning?  If a shadow is going to exercise the process on behalf of the lead, they will need full read/write access ahead of time.
+   * TODO: can the group name be identified here in a clear/transparent way?
+* Development client machine setup:
+   * ssh configuration set up for GitHub (either static .ssh/config or ssh agent works)
+   * run `gcloud init` using the identity that you had authorized in the prior bullet point
+   * git clone and checkout the [k/release repo](https://github.com/kubernetes/release) for the `gcbmgr` and `release-notify` scripts
+   * install [Google Cloud SDK](https://cloud.google.com/sdk/install)
+   * TODO: is there a preference here for Linux over MacOS/Windows, or should any gcloud and Kubernetes development supported platform work?
+   * ability to run `sendmail` from local Unix command line (BUG: should release-notify script be sending from GCB instead of the local machine, and can it?)
+   * branchff: requires ability for your user to write /usr/local/ directory (BUG),
+     sudo priv's, and membership in https://github.com/orgs/kubernetes/teams/kubernetes-release-managers
+
+# Safety Check
+Run the following command to affirm your release repo checkout is in a good state, your development client is fully configured, and your user identity is authorized for the builds:
+* `./gcbmgr`
+   * should run successfully and even show some green colored "OK" words
+* `./gcbmgr --project kubernetes-release-test container builds list`
```

Comment:
Maybe it should be `gcloud --project kubernetes-release-test builds list`?

Reply:
Updated.